Sources/GRPC/GRPCChannel/EmbeddedGRPCChannel.swift

```diff
@@ -1,113 +0,0 @@
-/*
```

Comment:
This file was only used in tests, its new home is in `Tests/GRPCTests/ClientTimeoutTests.swift`.